bench/bench_nlp_constraints.md

```diff
@@ -4,29 +4,30 @@
 using BenchmarkTools
```

Comment:
@BaptisteCbl just for the record: can you please recall me what these `bench` folder and `bench_nlp_constraints` are for? (did not remember this stuff.)

Reply:
The bench folder contains programs to generate a markdown file with the performance of the Julia file. Apart from the examples that use CTBase, it is independent of CTBase. I don't think it was meant to stay here. But it was actually developed by @ocots, he may have a better answer.